prody/proteins/interactions.py

```diff
@@ -3786,3 +3881,154 @@ def parseLigandInteractions(self, filename):
         
         return data
 
+
+    def getInteractionTypes(self):
+        """Show which interaction types were detected for ligands."""
+        
+        interactions = self._interactions_traj 
+        unique_keywords = set()
+
+        for sublist in interactions:
+            for sublist_item in sublist:
+                keyword = sublist_item[0]
+                unique_keywords.add(keyword)
+
+        unique_keywords_list = list(unique_keywords)
+        LOGGER.info("Interaction types: {0}".format(unique_keywords_list))
+        
+        return unique_keywords_list
+
+
+    def getLigandsNames(self):
+        """Show which ligands are in a system."""
+        
+        interactions = self._interactions_traj 
+        ligands = set()
+
+        for sublist in interactions:
+            for sublist_item in sublist:
+                keyword = sublist_item[4]
+                ligands.add(keyword)
+
+        ligands_list = list(ligands)
+        
+        return ligands_list
+
+
+    def getFrequentInteractors(self, **kwargs):
+        """Provide a dictonary with residues involved in the interaction with ligand
+        and their number of counts. 
+
+        :arg selection: selection string of ligand with chain ID
+                        e.g. MESA where MES is ligand resname and A is chain ID.
+        :type selection: str
+
+        :arg contacts_min: Minimal number of contacts which residue may form with ligand.
+                            by default is 2. 
+        :type contacts_min: int  """
+
+        atoms = self._atoms   
+        interactions = self._interactions_traj
+        contacts_min = kwargs.pop('contacts_min', 2)
+        selection = kwargs.pop('selection', None)
+        
+        from collections import Counter
+
+        if selection == 'total':  # Compute all interactions without distinguishing ligands
```

Comment:
should this be `None`, which is the default? Otherwise, you need to document about using 'total'

Reply:
Now it is None and I added more information about that.